.travis.yml

```diff
@@ -65,14 +58,17 @@ jobs:
         - pip install --user --upgrade pip
         - pip install --user yamllint
         # install golangci-lint
+        - export GOLANGCI_VERSION=$(source build.env ; echo ${GOLANGCI_VERSION})
```

Comment:
@nixpanic cant we source a file in a single place here instead of differently doing it in each iterations?

Reply:
I was hoping so, but I could not find how to do that nicely in travis. After sourcing, the variables need to be exported so that child processes of the shell can consume them.

Listing them explicitly also makes it easier to find problems.